pages/common/pdftotext.md

```diff
@@ -0,0 +1,19 @@
+# pdftotext
+
+> Convert PDF files to plain text format.
+
+- Convert filename.pdf to plain text and save it as filename.txt:
+
+`pdftotext {{filename.pdf}}`
+
+- Convert pdf-file.pdf to plain text and save it as text-file.txt:
+
+`pdftotext {{pdf-file.pdf}} {{text-file.txt}}`
+
+- Convert pdf-file.pdf to plain text and print it to standard output:
+
+`pdftotext {{pdf-file.pdf}} -`
+
+- Convert pages 2, 3 and 4 of pdf-file.pdf to plain text and print them to standard output (parameters -f and -l can also be used with previous variants):
```

Comment:
Use this as a save to text file example. And no need to mention that `-f` and `-l` can be used with previous examples. That is implicit and in-fact how our pages are structured.

And does the command support expanded options for `-f` and `-l` ?

Reply:
No expanded options, just the short ones.